src/theme/SearchBar/index.js

```diff
@@ -214,8 +217,9 @@ function DocSearch({ contextualSearch, externalUrlRegex, ...props }) {
           crossOrigin='anonymous'
         />
         {/* This loads libraries for clym privacy popups. */}
-        <script src="https://widget.clym-sdk.net/blocking.js"></script>
-        <script src="https://widget.clym-sdk.net/clym.js"></script>
+        <script src="https://widget.clym-sdk.net/blocking.js"></script> 
+        <script>{`(function(d,s,i,w,o){ var js,cjs=d.getElementsByTagName(s)[0]; if(d.getElementById(i))return; js=d.createElement('script'); js.id=i; js.src='https://widget.clym-sdk.net/clym.js';js.onload=function(){Clym&&Clym.load(i,w,o)};cjs.parentNode.insertBefore(js, cjs) }(document,'script','clym-privacy','efccb797d8fa43acba85c4d5dutupsek',{}))`}</script>
```

Comment:
@HugoJLFerreira out of curiosity, why are we doing this in the SearchBar component rather than the main home page?

Reply:
search is available in every page, and its part of the header was already injecting code into the head segment via import Head from '@docusaurus/Head';
as the links are present in the footer if we were to put this on the main page only the clym links would not work on the normal documentation pages